src/openrpc/device.json

```diff
@@ -793,16 +834,115 @@
       },
       "AudioProfiles": {
         "title": "AudioProfiles",
-        "allOf": [
-          {
-            "$ref": "https://meta.comcast.com/firebolt/types#/definitions/BooleanMap"
+        "type": "object",
+        "properties": {
+          "stereo": {
+            "type": "boolean"
           },
-          {
-            "type": "object",
-            "propertyNames": {
-              "$ref": "https://meta.comcast.com/firebolt/types#/definitions/AudioProfile"
-            }
+          "dolbyDigital5.1": {
+            "type": "boolean"
+          },
+          "dolbyDigital5.1+": {
+            "type": "boolean"
+          },
+          "dolbyAtmos": {
+            "type": "boolean"
+          }
+        },
+        "required": [
+          "stereo",
+          "dolbyDigital5.1",
+          "dolbyDigital5.1+",
+          "dolbyAtmos"
+        ]
+      },
+      "HDRFormat": {
+        "title": "HDRFormat",
+        "type": "object",
+        "properties": {
+          "hdr10": {
+            "type": "boolean"
+          },
+          "hdr10Plus": {
+            "type": "boolean"
+          },
+          "dolbyVision": {
+            "type": "boolean"
+          },
+          "hlg": {
+            "type": "boolean"
           }
+        },
+        "required": [
+          "hdr10",
+          "hdr10Plus",
+          "dolbyVision",
+          "hlg"
+        ],
+        "description": "The type of HDR that is supported"
+      },
+      "HDCPVersion": {
```

Comment:
Please change this to `HDCPVersionMap` to differentiate it from a future `HDCPVersion` object.

Same for `HDRFormat` to `HDRFormatMap`.